.cirrus.yml

```
@@ -109,7 +109,7 @@ extension_smoketest_task:
     - mkdir tmp && cd tmp
     - mkdir testextension && cd testextension
     - pwd
-    - python3 -m cryptoadvance.specter ext gen --ext-id cicdtest --org cryptoadvance --no-isolated-client --devicename cicddevice
+    - python3 -m cryptoadvance.specter ext gen --ext-id cicdtest --org cryptoadvance --no-isolated-client --no-encrypted-userdata --devicename cicddevice --version v2.0.0
```

I think the version should be omitted here.
```suggestion
    - python3 -m cryptoadvance.specter ext gen --ext-id cicdtest --org cryptoadvance --no-isolated-client --no-encrypted-userdata --devicename cicddevice
```
We want to know whether the current branch this is currently being tested, works on the master of `specterext-dummy`. That might be not optimal in some cases but the best we can get for most of the cases where we have chages in a PR which (hopefully) still match the state in master of `specterext-dummy`.

---

the smoke test fails if I remove this

---

Okay, I read the issue again and I think, we need a version since templates are becoming more dependent on the version of Specter controlling the generation process and if no version is specified, it should fall back to master.

If the user chooses to add a specific version, it will render extension accordingly, else the tag(branch) will be master and version will be the latest one.